src/main/java/tennisclub/entity/Lesson.java

```diff
@@ -0,0 +1,69 @@
+package tennisclub.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.ManyToMany;
+import javax.persistence.OneToMany;
+import java.time.LocalDateTime;
+import java.util.Collections;
+import java.util.Set;
+
+/**
+ * @author Xuan Linh Phamová
+ */
+@Entity
+public class Lesson extends Event{
+    private int capacity;
+
+    @ManyToMany(mappedBy = "lessonsToTeach")
+    private Set<User> teachers;
```

Comment:
Should there not be only 1 teacher for every lesson (=> probably `@ManyToOne` used here)? The cardinality in the class diagram seems to suggest that.

Reply:
Sorry, did not notice the change in the class diagram (next time, I will pull the branch and look at the changed PNGs locally). Furthermore, multiple teachers per lesson make sense. Let us keep it this way. :smile:

Reply:
Sorry, did not notice the change in the class diagram (next time, I will pull the branch and look at the changed PNGs locally). Furthermore, multiple teachers per lesson make sense. Let us keep it this way. :smile: